website/docs/typescript.mdx

```diff
@@ -10,8 +10,7 @@ sidebar_label: TypeScript / Flow
 		data-ea-type="image"
 		className="horizontal bordered"
 	></div>
-</center>
-<details>
+</center> <details>
```

Comment:
I actually have no idea what this is, gitpod shows this line as not changed:
![image](https://user-images.githubusercontent.com/2545357/113717696-dfa23d80-96b9-11eb-9acb-bfb37d06340f.png)

Auto formatter?

Reply:
looks like a different line ending. Probably it's fine :)